AsterNET.ARI/StasisEndpoint.cs

```diff
@@ -8,22 +8,30 @@ public class StasisEndpoint
         /// <param name="port"></param>
         /// <param name="username"></param>
         /// <param name="password"></param>
```

Comment:
again, can we add param docs for new parameters please

Reply:
will do